src/System.Private.Jit/src/System.Private.Jit.csproj

```diff
@@ -66,6 +66,7 @@
     <Compile Include="$(CommonBasePath)\Internal\Text\Utf8String.cs" />
     <Compile Include="$(CommonBasePath)\Internal\Text\Utf8StringBuilder.cs" />
     <Compile Include="$(CommonBasePath)\System\Collections\Generic\ArrayBuilder.cs" />
+    <Compile Include="$(TypeSystemBasePath)\Mangling\IPrefixMangledType.cs" />
```

Comment:
Please make `NativeStructType` a partial class so that this is not necessary. JIT should not carry bloat that deals with name mangling.